src/main/resources/querydsl_checks_base.xml

```diff
@@ -72,7 +69,7 @@
     <module name="WhitespaceAround"/>
 
     <module name="ModifierOrder"/>
-    <module name="RedundantModifier"/>
+<!--    <module name="RedundantModifier"/>-->
 
```

Comment:
Ah yes, I see I still had to reapply a few rules here. Somehow they started giving issues after the update.

Should we first merge this and then update the other PR?

Reply:
Yes, this first, the other will be simpler to resolve